packages/react-components/react-progress/stories/ProgressBar/ProgressBarBestPractices.md

```diff
@@ -1,6 +1,6 @@
 <details>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30658&commitId=ad13930ff5d6391f1061b31ec8e8163ee53ded5f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+image+badge.normal.chromium.png|725| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
clearly not related